SwanContents/swancontents/swanclassic/handlers/projects.py

```diff
@@ -59,9 +73,9 @@ def get(self, path=''):
                                             server_root=self.settings['server_root_dir'],
                                             projects_page=True,
                                             ))
-        elif cm.file_exists(swan_path):
+        elif file_exists:
             # it's not a directory, we have redirecting to do
-            model = cm.get(swan_path, content=False)
+            model = yield ensure_async(cm.get(swan_path, content=False))
```

Comment:
Why do you need to make the `get` a coroutine and yield on it?

Reply:
https://www.tornadoweb.org/en/stable/guide/coroutines.html#native-vs-decorated-coroutines

I kept the old style in nbclassic handlers (they also use it), but actually, it would be okay to use await everywhere, as we only support Python 3.9... In some places I had even messed up... I'll update everywhere (only using async await).

Reply:
Yes let's use native coroutines everywhere!